build/rbac/rbac.yaml

```diff
@@ -551,6 +587,12 @@ metadata:
   labels:
     operator: rook
     storage-backend: ceph
+    app.kubernetes.io/name: rook-ceph
+    app.kubernetes.io/component: csi
+    app.kubernetes.io/part-of: rook-ceph
+    app.kubernetes.io/managed-by: Helm
+    app.kubernetes.io/created-by: rook-ceph-operator
+    helm.sh/chart: rook-ceph-0.0.1
```

Comment:
How about we not include version info here? They can look at the helm chart version instead of the version labels on the individual resources